### PR TITLE
List of Tasks implemented

### PR DESCRIPTION
**Pull request by SwethaJanardhana**

### CODE REVIEW

**Review by onemanwenttomow — APPROVED** [review]
Nice work team. LGTM 🎉

File: pages/_app.js [hunk 1]
```diff
@@ -5,10 +5,15 @@ import { useState } from "react";
 export default function App({ Component, pageProps }) {
   const [tasks, setTasks] = useState(initialTasks);
 
+  // Sorting the task in chronological order of date
+  const tasksAfterSorting = tasks.sort(
```
Comment (FelineHuhn) [inline pages/_app.js:8]: What about just `sortedTasks`? :)

File: pages/_app.js [hunk 2]
```diff
@@ -1,10 +1,19 @@
 import GlobalStyle from "../styles";
+import initialTasks from "@/db/lib/tasks";
+import { useState } from "react";
 
 export default function App({ Component, pageProps }) {
+  const [tasks, setTasks] = useState(initialTasks);
```
Comment (FelineHuhn) [inline pages/_app.js]: Why you already implement the `useState` hook for this feature? You don't use the setter function yet, so you should go the easiest way by just sorting the initialTasks you import. You only need useState when you add tasks.

File: pages/_app.js [hunk 3]
```diff
@@ -1,10 +1,18 @@
 import GlobalStyle from "../styles";
+import initialTasks from "@/db/lib/tasks";
 
 export default function App({ Component, pageProps }) {
+  const tasks = initialTasks;
```
Comment (FelineHuhn) [inline pages/_app.js:5]: I don't think that you need this step and could directly sort the initialTasks in line 8. But if you find it more legible this way, leave it that way.

**Review by FelineHuhn — APPROVED** [review]
Great first user story! Congrats team 🤩🥳 Left you a small comment but my suggestion is a matter of taste I think :)

**Pull request merged**